docs/en/patterns/tabs.md

```diff
@@ -5,20 +5,34 @@ table_of_contents: true
 
 ## Tabs
 
-Use tabs when there are multiple categories/views/panes of content, but there is the need to only show one pane at a time.
+<hr>
 
-Don’t use tabs for pagination of content or cases involve viewing content, not navigating between groups of content. Tabs can not be nested
-or be mixed with icons or button styles.
+Tabs organise and allow navigation between groups of content that are related and at the same level of hierarchy.
 
 To select the active tab add the attribute `aria-selected="true"` and that list item will have the correct styling.
 
 <a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/tabs/"
     class="js-example">
-    View example of the tabs pattern
+View example of the tabs pattern
 </a>
 
-<hr />
+### Usage
+
+<div class="row">
+   <div class="col-6">
+   <div class="p-notification--positive">
+   <p class="p-notification__response"><span class="p-notification__status">Do:</span>Use when there are multiple categories, views or panes of content.</p>
+   </div>
+   <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/252d5420-navigation-tabs-color-do.png" alt="Tabs do">
+   </div>
+  <div class="col-6">
+  <div class="p-notification--negative">
+  <p class="p-notification__response"><span class="p-notification__status">Don't:</span>Mix tabs that contain only text, with tabs that contain only icons.</p>
```

Comment:
Is this meant to say "...with tabs that contain ~~only~~ icons"?

Reply:
@good spot. Will update 👍